kernelci/kbuild.py

```diff
@@ -699,16 +699,17 @@ def submit(self, retcode, dry_run=False):
             },
             'child_nodes': [],
         }
-        # TODO: Fix this hack
+        # Update node from API, as we might have new fields
+        # such as k8s_context
+        nodeid = self._node['id']
```

Comment:
For better readability,
```suggestion
        node_id = self._node['id']
```